bin/bossfetch

```diff
@@ -119,6 +121,26 @@ def main():
         print('No files to fetch.')
         return 0
 
+    if args.globus:
```

Comment:
This only makes sense if `args.save` is also set so perhaps exit with error unless this is True?